notebook/nbextensions.py

```diff
@@ -180,7 +180,7 @@ def install_nbextension(path, overwrite=False, symlink=False,
         if not destination:
             destination = basename(path)
         destination = cast_unicode_py2(destination)
-        full_dest = pjoin(nbext, destination)
+        full_dest = normpath(pjoin(nbext, destination))
```

Comment:
we don have logging in https://github.com/malev/notebook/blob/fix-symlink/notebook/nbextensions.py#L196